schemas/performance-artifact.json

```diff
@@ -80,7 +80,7 @@
           "title": "Extra options used in running suite",
           "items": {
             "type": "string",
-            "maxLength": 45
+            "maxLength": 100
```

Comment:
Looking closer over this PR, I realized we still need an extra change. More precisely, a schema migration [here](https://github.com/mozilla/treeherder/blob/master/treeherder/perf/models.py#L54).

The current VARCHAR length of the equivalent `extra_options` database column == [suite_schema.extraOptions.items.maxLength](https://github.com/mozilla/treeherder/blob/master/schemas/performance-artifact.json#L83) * [suite_schema.extra_options.maxItems](https://github.com/mozilla/treeherder/blob/master/schemas/performance-artifact.json#L86). Which is 360 == 45 * 8.

Because we need to increase suite_schema.extraOptions.items.maxLength to 100, that would mean that we should increase VARCHAR to 800 == 100 * 8 as well.

Reply:
That, I don't know - I'm pretty far out of my sphere here!  I'd leave the choice to your discretion.

Another option might be to try to "summarize" the machine types in-tree somehow, maybe something as simple as `mtype[-45:]` or perhaps a hash.  It really depends on how useful that data is to performance analysis.